pyproject.toml

```diff
@@ -32,7 +32,7 @@ dependencies = [
     "nilearn == 0.10.1",
     "nipype == 1.9.1",
     "nireports >= 24.0.3",
-    "niworkflows >=1.9,<= 1.10",
+    "niworkflows >= 1.13,< 1.14",
```

Comment:
In the older pinned version, niworkflows' skeleton-generation function didn't support "extension."